app/workers/my_anime_list_sync_worker.rb

```diff
@@ -1,10 +1,20 @@
 class MyAnimeListSyncWorker
   include Sidekiq::Worker
 
-  def perform(library_entry_id, method)
+  # data is a hash
+  def perform(data)
```

Comment:
IDK man, this seems kinda janky.  Perhaps just pass in `user_id`, `anime_id` and then `library_entry_id` which can be a special value like `'delete'` to signify a deletion?

```ruby
def perform(user_id, media_type, media_id, library_entry_id)
  if library_entry_id == 'delete'
    # Delete it
  else
    # Sync it
  end
end
```

Params list is a bit long, but at least it's not an opaque hash